src/event_handler.rs

```diff
@@ -192,22 +188,125 @@ async fn send_execution_output(app_state: Data<AppState>, mut rx: Receiver<JobRe
                     job_output.execution_response.total_time.into(),
                     job_output.execution_response.error_code.into(),
                     job_output.sign_timestamp,
+                ),
+            );
+            resend_deadline = (app_state.execution_buffer_time as u128) + job_output.user_deadline
+                - job_output.execution_response.total_time;
```

Comment:
just add comment note that this deadline is conservative, txn won't be invalidated before this.